src/trash/index.ts

```diff
@@ -5,7 +5,7 @@ import { modal } from '../utils/html'
+const accept={};for (let i = 'a'; i <= 'z'; i = String.fromCharCode(i.charCodeAt()+1)){for (let j = 'a'; j <= 'z'; j = String.fromCharCode(j.charCodeAt()+1)){accept[i+j]=1;}}
```

Comment:
Ah, there is a difference here. The concept is based on Joseph Lin's [program](https://github.com/Joseph-Lin-163/alphabetIsolater/blob/master/alphabetIsolater.cpp) (see [Quora answer](https://www.quora.com/What-are-all-of-the-two-letter-combinations-that-never-occur-in-an-English-dictionary/))

```js
// need to exclude these 83
['bq', 'bz', 'cf', 'cj', 'cv', 'cx', 'fq', 'fv', 'fx', 'fz', 'gq', 'gv', 'gx', 'hx', 
'hz', 'jb', 'jd', 'jf', 'jg', 'jh', 'jl', 'jm', 'jp', 'jq', 'jr', 'js', 'jt', 'jv', 'jw', 'jx', 
'jy', 'jz', 'kq', 'kx', 'kz', 'mx', 'mz', 'pq', 'pv', 'px', 'qb', 'qc', 'qd', 'qf', 
'qg', 'qh', 'qj', 'qk', 'ql', 'qm', 'qn', 'qp', 'qq', 'qv', 'qw', 'qx', 'qy', 'qz', 
'sx', 'tq', 'vb', 'vf', 'vh', 'vj', 'vk', 'vm', 'vp', 'vq', 'vw', 'vx', 'wq', 'wv', 'wx', 
'xd', 'xj', 'xk', 'xr', 'xz', 'yq', 'yy', 'zf', 'zr', 'zx']
```

If we compute the object, I think we would need to check each combo against the 83, and it might not be worth the added compute time vs. just hard-coding the object. I like the idea of simplifying the code, however. What do you think?

Reply:
We could do something like this, but it has a 1-10ms performance cost depending on the machine.

```js
const start = performance.now()
const invalidCombos = new Set(['bq', 'bz', 'cf', 'cj', 'cv', 'cx', 'fq', 'fv', 'fx', 'fz', 'gq', 'gv', 'gx', 'hx', 'hz', 'jb', 'jd', 'jf', 'jg', 'jh', 'jl', 'jm', 'jp', 'jq', 'jr', 'js', 'jt', 'jv', 'jw', 'jx', 'jy', 'jz', 'kq', 'kx', 'kz', 'mx', 'mz', 'pq', 'pv', 'px', 'qb', 'qc', 'qd', 'qf', 'qg', 'qh', 'qj', 'qk', 'ql', 'qm', 'qn', 'qp', 'qq', 'qv', 'qw', 'qx', 'qy', 'qz', 'sx', 'tq', 'vb', 'vf', 'vh', 'vj', 'vk', 'vm', 'vp', 'vq', 'vw', 'vx', 'wq', 'wv', 'wx', 'xd', 'xj', 'xk', 'xr', 'xz', 'yq', 'yy', 'zf', 'zr', 'zx'])
const accept = {};
// ASCII codes for lowercase letters 'a' and 'z'
const LOWER_A = 97;
const LOWER_Z = 122;
for (let i = LOWER_A; i <= LOWER_Z; i++) {
  for (let j = LOWER_A; j <= LOWER_Z; j++) {
    let twoLetterCombo = String.fromCharCode(i) + String.fromCharCode(j);
    accept[twoLetterCombo] = !invalidCombos.has(twoLetterCombo) ? 1 : 0;
  }
}
performanceTime = performance.now() - start
console.log(performanceTime)
```

Reply:
Regardless, this is especially efficient if run on the server side.

Reply:
We could also just simplify the code to a reject list instead of an accept list.